<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RereferenceWidget</class>
 <widget class="QWidget" name="RereferenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxModality">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Modality</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_EEG">
        <property name="text">
         <string>EEG</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupModality</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_MEG">
        <property name="text">
         <string>MEG</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupModality</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_EMEG">
        <property name="text">
         <string>EEG + MEG</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupModality</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBoxMethod">
     <property name="title">
      <string>Method</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_CAR">
        <property name="text">
         <string>CAR</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupMethod</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_Selection">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Selection</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupMethod</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_File">
        <property name="text">
         <string>File</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupMethod</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_File">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_File">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_Enabled">
     <property name="text">
      <string>Enable Rereferencing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupMethod"/>
  <buttongroup name="buttonGroupModality"/>
 </buttongroups>
</ui>
